pandas/tests/series/test_operators.py

```diff
@@ -962,6 +962,14 @@ def test_timedelta64_ops_nat(self):
 
 
 class TestDatetimeSeriesArithmetic(object):
+    @pytest.mark.xfail(reason='GH#18824 bug in Timedelta.__floordiv__')
```

Comment:
is this the right issue?

Reply:
AFAIK there is no dedicated issue.  I made a checkbox for this in 18224.

Reply:
don't mark 18224, its way to big to figure out what you are talking about (you can certainly *reference* this PR which is I think what you are doing). rather list the PR number (if there isn't an issue). do this generally.